docker-compose.yml

```diff
@@ -7,6 +7,10 @@ services:
       - '9090:8080'
     volumes:
     - ./target/scala-2.12/hmda.jar:/opt/hmda.jar
+    depends_on:
+      - query_db
+    environment:
+      JDBC_URL: jdbc:postgresql://192.168.99.100:54321/hmda?user=postgres&password=postgres
```

Comment:
This will be more portable if you reference the service name and container port.  I _think_ the following should work:

```yaml
JDBC_URL: jdbc:postgresql://query_db:5432/hmda?user=postgres&password=postgres
```